_docs/bigquery/enabling-billing.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+section: bigquery
+navigation_title: "Enabling billing"
+title: "Enabling billing"
+order: 3
+date: 2020-04-29
+---
+
+<!---
+In this article explained how to Enable BigQuery API if it is not enabled
+-->
+## Before you begin
+* You should have an existing [Google Cloud Platform account ▸](creating-gcp-account)
+
+To use BigQuery with SegmentStream you need an active billing account so that you'll be able to use Google Cloud Platform resources.
+
+**Note: Free Tier Google Cloud accounts don’t support some BigQuery services which are required for SegmentStream Data Import you need to upgrade your account**
+
+## Check if you have an active Free Tier account, and upgrade it if possible:
```

Comment:
Header can't have ":", "," and instructions. Header should be header:

#Upgrading a Free Tier account